optimade/server/config.py

```diff
@@ -280,6 +280,10 @@ class ServerConfig(BaseSettings):
         True,
         description="If True, the server will check whether the query parameters given in the request are correct.",
     )
+    enabled_response_formats: Optional[List[str]] = Field(
```

Comment:
Should make an enum of supported formats, then do `Optional[List[SupportedFormats]]` like some of the other options

Reply:
Ok, I am trying to do this, but it does make things more complicated because I now have to convert the enums to a string before I can do the comparisons in my code. It would be easier to use a Literal["json", "hdf5"] instead.

Reply:
I am now using an ENUM class to restrict which values can be specified for enabled_response_formats.